### PR TITLE
[C-6480] Audit Events

### PR DESCRIPTION
**Pull request by tk26**
## Description of the change

Audit Events API

Screen Shot 2022-07-25 at 7 52 21 PM

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/main/java/models/AuditEvent.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@Getter
+@Setter
+@ToString
+public class AuditEvent {
+    public String auditEventId;
+    public String source;
+    public String timestamp;
+    public String type;
+    public String workspaceId;
+    public Object actor;
```
Comment (ryanknapp) [inline src/main/java/models/AuditEvent.java:16]: Returning Object for actor and target is kind of useless. Do we have proper type definitions for these?

File: src/main/java/models/AuditEvent.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@Getter
+@Setter
+@ToString
+public class AuditEvent {
+    public String auditEventId;
+    public String source;
+    public String timestamp;
+    public String type;
+    public String workspaceId;
```
Comment (scarney81) [inline src/main/java/models/AuditEvent.java:15]: Why are we exposing `workspaceId`? This is not a concept for our customers

**Review by scarney81 — CHANGES_REQUESTED** [review]
We should not be exposing workspaceId publicly

File: src/main/java/models/AuditEvent.java (hunk 1 above)
Comment (tk26) [reply src/main/java/models/AuditEvent.java]: good call. we have id and email right now! Added classes for actor and target!

File: src/main/java/models/AuditEvent.java (hunk 2 above)
Comment (tk26) [reply src/main/java/models/AuditEvent.java]: Fixed!

**Review by ryanknapp — APPROVED** [review]

**Review by scarney81 — APPROVED** [review]

**Pull request merged**